File: benchmark/corpus/10_control/f_math_in_loop.py
# Sums square roots in a counted loop through the host's math library, and declares ffi:math.
import math


def sum_of_roots(n):
    total = 0.0
    i = 1
    while i <= n:
        total = total + math.sqrt(i)
        i = i + 1
    return total


print("about %d" % round(sum_of_roots(10) * 1000.0))

File: benchmark/corpus/10_control/f_math_in_loop.js
// Sums square roots in a counted loop through the host's math library, and declares ffi:math.
function sumOfRoots(n) {
  let total = 0.0;
  let i = 1;
  while (i <= n) {
    total = total + Math.sqrt(i);
    i = i + 1;
  }
  return total;
}

console.log("about " + Math.round(sumOfRoots(10) * 1000.0));
